xtask/src/codegen/aya.rs

```diff
@@ -211,7 +217,10 @@ fn codegen_bindings(opts: &SysrootOptions, libbpf_dir: &Path) -> Result<(), anyh
                 .constified_enum("BTF_VAR_.*")
                 .constified_enum("IFLA_.*")
                 .constified_enum("TCA_.*")
-                .constified_enum("BPF_RINGBUF_.*");
+                .constified_enum("BPF_RINGBUF_.*")
```

Comment:
```suggestion
```
Looks like this is a duplicate of the line above?

Reply:
or line below even - either way I'm seeing 2 of them lol

Reply:
ahh, thanks for catching that 😵